superset-frontend/src/explore/components/controls/VizTypeControl.jsx

```diff
@@ -18,15 +18,16 @@
  */
 import React, { useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
-import { Row, Col, FormControl } from 'react-bootstrap';
+import { Input } from 'src/common/components';
+import { Row, Col } from 'react-bootstrap';
```

Comment:
I guess a note to change here to move Row Col to use ant-d. This might of been touched up already from the row/col migration

Reply:
Yes, this was resolved in https://github.com/apache/superset/pull/14100 already.